react-context/client/src/components/NavBar.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useContext } from 'react';
+import { Link, Outlet } from 'react-router-dom';
+import { FaBolt, FaSignOutAlt } from 'react-icons/fa';
+import { AppContext } from './AppContext';
+
+export function NavBar() {
+  /* TODO: Grab `user` and `handleSignOut` from `AppContext` */
+  const context = useContext(AppContext);
```

Comment:
The nicest way to do these is with destructuring:
`const { user, handleSignOut } = useContext(AppContext);`